.bazelci/presubmit.yml

```diff
@@ -100,7 +100,7 @@ tasks:
     # TODO: Change to "rolling" once
     # https://github.com/bazelbuild/bazel/commit/f3aafea59ae021c6a12086cb2cd34c5fa782faf1
     # is available in rolling.
-    bazel: "last_green"
+    bazel: "2b8219042c132483e0af39ef20d67dfd6442af01"
```

Comment:
Please put a comment to link to the issue about why a particular commit is chosen